termine desafio 3

ThomyQ1 opened the pull request:
desafio 3 finalizado
<img width="368" alt="users id" src="https://github.com/ThomyQ1/BackendCourse/assets/138169226/72be0e62-8207-408f-b341-bdda12dd320a">
<img width="387" alt="users al" src="https://github.com/ThomyQ1/BackendCourse/assets/138169226/9b4c24ee-0d53-46cb-9b4d-d45e04f87b21">
<img width="496" alt="products al" src="https://github.com/ThomyQ1/BackendCourse/assets/138169226/d7e5944a-9d9a-4c41-893a-99d6fa3759d5">
<img width="584" alt="products id" src="https://github.com/ThomyQ1/BackendCourse/assets/138169226/2a41ac59-d861-491d-9ecb-d09bbd124061">

ThomyQ1 committed:
termine desafio 3

desafio 3 finalizado

ThomyQ1 commented:
termine desafio 3

jptuttolomondo commented:
Muy bien!
Muy bien la implementación de laas clases, rutas, server.

Elimina los console.log innecesarios.

Subi captura de pantallas de las respuestas de los endpoints.

ThomyQ1 committed:
Cosas del desafio 3 acomodadas y mejor realizadas.

Modifique varias cosas para fomentar un mejor codigo de desarrollo sobre el servidor
